app/keepers/keepers.go

```diff
@@ -6,9 +6,9 @@ import (
 	// unnamed import of statik for swagger UI support
 	_ "github.com/cosmos/cosmos-sdk/client/docs/statik"
 
-	"github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v4/router"
-	routerkeeper "github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v4/router/keeper"
-	routertypes "github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v4/router/types"
+	"github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v4/packetforward"
```

Comment:
can you alias this as router so you don't need to change anything else aside from the import?
```suggestion
	router "github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v4/packetforward"
```